### DISCUSSION TITLE
How to use Android TV?

### DISCUSSION THREAD

**Discussion by proprivacyio** [fast4x/RiMusic#1427] [Q&A]
Hi! Absolutely love the app and I see it has Android TV support. I have an Nvidia Shield. How can I get to stream music from RiMusic to Android TV? 

Thanks!

**Comment by proprivacyio** [comment]
I am a user of GrapheneOS by the way. Is there any other way to stream RiMusic to my Hifi set, maybe with an audiostreamer? https://iear.nl/audio-en-hifi/audio-bronnen/audio-streamer/

**Comment by peat80** [comment]
Sideload it on your shield and try.
Not every button is reachable and some things are hard to navigate with remote but most things do work.

I would sugesst to install it on your phone, follow artists and things you like there and then export the database to your tv for a good starting point. 😉
